src/App.js

```diff
@@ -1,26 +1,45 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Axios from 'axios';
+import FriendsList from './components/FriendsList';
+import FriendsListForm from './components/FriendsListForm';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: []
+    }
+  }
+
+  componentDidMount() {
+    Axios
```

Comment:
Pull out the logic in componentDidMount() and put in a seperate function in App.js.,
```resetFriendsState = () => { logic }```, and then call the function in componentDidMount, ```componentDidMount() { this.resetFriendsState}```.  With resetFriendsState being an App.js function, we can pass it to the FriendsListForm component, and fire off the function after every successful post or put request. This is necessary because otherwise the parent component wouldn't be aware of the change in state due to the post or put requests in the child comp caused (this true state is held in the back-end. this is how we persist data (i.e. save data)).